README.md

```diff
@@ -27,6 +27,7 @@ WORKSPACE=### Set to `dev` for local development, this will be set to `stage` an
 ```shell
 S3_RESTRICTED_CDN_ROOT=### S3 bucket + prefix for CDN restricted, e.g. 's3://<bucket>/path/to/restricted'
 S3_PUBLIC_CDN_ROOT=### S3 bucket + prefix for CDN public, e.g. 's3://<bucket>/path/to/public'
+S3_TIMDEX_ALMA=### S3 bucket + prefix for previously extracted Alma source records, e.g. 's3://timdex-extract-<env>-<arn>'
```

Comment:
I might suggest the actual prod S3 URI we'd expect, e.g.

```
s3://<TIMDEX_EXTRACT_BUCKET>/alma
```

Reply:
Updated to `s3://<timdex-extract-bucket>/alma` to match syntax of other descriptions!